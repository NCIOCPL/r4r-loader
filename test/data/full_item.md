---
id: 9999
title: Full Item Title
description: >
    Etiam cuncti quaque
    praefixaque plura dorso hac.
website: https://www.example.org/item
toolTypes:
  - toolType: datasets_databases/clinical_data
  - toolType: datasets_databases/epidemiologic_data
researchAreas:
  - researchArea: cancer_biology
  - researchArea: cancer_omics
researchTypes:
  - researchType: basic
  - researchType: clinical
resourceAccess:
  type: open
  notes: This resource is free like it says
docs:
  - doc: oet
poc:
  - email: john.doe@example.org
    name:
      prefix: Dr.
      firstname: John
      middlename: Alexander
      lastname: Doe
      suffix: Ph.D.
    phone: 240-555-1212
    title: Program Manager NCI CBIIT
---
## Promissa ignavos putate

Lorem markdownum submovet dabant; tum suaque neci [gerens at
Lycabas](http://mihi-pectora.net/) videat *mihi*? Columbas femina: adulter
prohibent aeternas dicitur traxere silvas suscipitur favilla, suo nobis, erigor
inopes, luctusque rursus. Ab tollere partes tu hiatus [Troiana
sic](http://www.iussit.org/in-erat.php) aegros *magno et orbe* coniuge patriam
velamina.

- Maius inops ominibus in tamen continet tibi
- Orbis nox deorum supplex sententia domitamque Troiae
- In veluti sanguine melior
- Intellegit pervenit actum ipse antrum ingenium et
- Timendos comitem que et Cyclopis in intravit
- Lintea esse sollicita caecaque procul