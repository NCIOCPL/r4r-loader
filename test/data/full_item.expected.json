{
  "id": 9999,
  "title": "Full Item Title",
  "description": "<p>Etiam cuncti quaque praefixaque plura dorso hac.",
  "website": "https://www.example.org/item",
  "toolTypes": [
    {
      "type": { "key": "datasets_databases", "label": "Datasets & Databases" },
      "subtype": { "key": "clinical_data", "label": "Clinical Data" }
    },
    {
      "type": { "key": "datasets_databases", "label": "Datasets & Databases" },
      "subtype": { "key": "epidemiologic_data", "label": "Epidemiologic Data" }
    },
    {
      "type": { "label": "Analysis Tools", "key": "analysis_tools" }
    }
  ],
  "researchAreas": [
    { "label": "Cancer Biology", "key": "cancer_biology" },
    { "label": "Cancer Omics", "key": "cancer_omics" }
  ],
  "researchTypes": [
    { "label": "Basic", "key": "basic" },
    { "label": "Clinical", "key": "clinical" }
  ],
  "resourceAccess": {
      "notes": "This resource is free like it says",
      "type": "open"
  },
  "docs": [
    { "label": "Office of Examples and Tests (OET)", "key": "oet" }
  ],
  "pocs": [
    {
      "email": "john.doe@example.org",
      "name": {
        "firstname": "John",
        "lastname": "Doe",
        "middlename": "Alexander",
        "prefix": "Dr.",
        "suffix": "Ph.D."
      },
      "phone": "240-555-1212",
      "title": "Program Manager NCI CBIIT"
    }
  ],
  "body": "<h2>Promissa ignavos putate</h2><p>Lorem markdownum submovet dabant; tum suaque neci <a href=http://mihi-pectora.net/>gerens at Lycabas</a> videat <em>mihi</em>? Columbas femina: adulter prohibent aeternas dicitur traxere silvas suscipitur favilla, suo nobis, erigor inopes, luctusque rursus. Ab tollere partes tu hiatus <a href=http://www.iussit.org/in-erat.php>Troiana sic</a> aegros <em>magno et orbe</em> coniuge patriam velamina.<ul><li>Maius inops ominibus in tamen continet tibi<li>Orbis nox deorum supplex sententia domitamque Troiae<li>In veluti sanguine melior<li>Intellegit pervenit actum ipse antrum ingenium et<li>Timendos comitem que et Cyclopis in intravit<li>Lintea esse sollicita caecaque procul</ul>"
}